sdk/metricsadvisor/azure-ai-metricsadvisor/src/main/java/com/azure/ai/metricsadvisor/models/AnomalyIncident.java

```diff
@@ -102,6 +129,25 @@ public DimensionKey getRootDimensionKey() {
         return this.rootDimensionKey;
     }
 
+    /**
+     * Gets the aggregated value at root dimension node where the incident anomalies
+     * are logically aggregated.
+     *
+     * @return The value.
+     */
+    public Double getValue() {
```

Comment:
Should dthis be primitive or can be nullable?

Reply:
It should be primitive (let me confirm), let's address that in a follow-up pr just to unblock us from adding the rest of the features.